hello/src/hello_backend_plugin.c

```diff
@@ -196,6 +256,11 @@ hello_validate(clicon_handle h, transaction_data td) {
     int rv;
 
     printf("*****hello validate*****\n");
+    printf("src:\n");
```

Comment:
This will show up on the backend's stdout. This is limiting to a debug case where you run the backend in the foreground.
I would add that as a log instead so you could steer the prints to syslog/stderr/stdout as described here: https://clixon-docs.readthedocs.io/en/latest/backend.html#command-line-options.
Logging XML can be done with the macro  clixon_log_xml(). Or debug with  clixon_debug_xml().